### PR TITLE
Remove decoration from forwarded exception

### PR DESCRIPTION
**Pull request by ro0NL**
See https://github.com/symfony/symfony/issues/22618

before
image

after
image

### CODE REVIEW

File: Composer/ScriptHandler.php [hunk 1]
```diff
@@ -290,7 +290,7 @@ protected static function executeCommand(Event $event, $consoleDir, $cmd, $timeo
         $process = new Process($php.($phpArgs ? ' '.$phpArgs : '').' '.$console.' '.$cmd, null, null, null, $timeout);
         $process->run(function ($type, $buffer) use ($event) { $event->getIO()->write($buffer, false); });
         if (!$process->isSuccessful()) {
-            throw new \RuntimeException(sprintf("An error occurred when executing the \"%s\" command:\n\n%s\n\n%s.", escapeshellarg($cmd), $process->getOutput(), $process->getErrorOutput()));
+            throw new \RuntimeException(sprintf("An error occurred when executing the \"%s\" command:\n\n%s\n\n%s.", escapeshellarg($cmd), self::removeDecoration($process->getOutput()), self::removeDecoration($process->getErrorOutput())));
```
Comment (stof) [inline Composer/ScriptHandler.php]: having the dot at the end of the message (alone on its own line) after the command output looks quite weird actually. I would remove it.
The command output is very likely to include sentences anyway.

**Comment by ro0NL** [comment]
Dot removed.

**Comment by ro0NL** [comment]
What about removing the `escapeshellarg` as well? It seems to cause double escaping + we're putting it between double quotes as well; removing it....

before
```
An error occurred when executing the "'assets:install --symlink --relative '\''web'\'''" command:
```

after
```
An error occurred when executing the "assets:install --symlink --relative 'web'" command:
```

Am i missing something?

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**